lua/orgmode/config/init.lua

```diff
@@ -42,6 +42,9 @@ function Config:extend(opts)
   self.opts = vim.tbl_deep_extend('force', self.opts, opts)
   if self.org_startup_indented then
     self.org_adapt_indentation = not self.org_indent_mode_turns_off_org_adapt_indentation
+    if self.org_indent_mode_turns_on_hiding_stars then
```

Comment:
Let's remove this and use all the checks only in the `stars` highlighter.

Reply:
Done as part of latest force push.